### ISSUE TITLE
Owl Nav not working properly

### ISSUE THREAD

**Issue by CracyCrazz** [OwlCarousel2/OwlCarousel2#1462]
When there are not enough items to fill the whole owl stage, but loop is true the navigation doesn't show.

Navigation should always show when it's set to true and loop is true as well.

**Comment by bpdinteractive** [comment]
I'm experiencing the owl-prev and owl-next buttons showing / hiding on every other slide. That is, every other slide will either show or hide BOTH buttons.

I suppose (with loop: false, anyway) that the first slide should show the owl-next div, slide 2 through N-1 show both divs, and only show owl-prev div on slide N.

**Comment by sunfirester** [comment]
I am seeing the same hiding/showing of the nav text.

**Comment by PeteFromSales** [comment]
Has this bug been fixed yet? I'm on 2.2.1, and when I have `settings.nav: true` and `settings.loop: true`, `owl-nav` remains disabled, seemingly due to:

```javascript
var difference,
        settings = this._core.settings,
        disabled = this._core.items().length <= settings.items && !settings.loop,
        index = this._core.relative(this._core.current()),
        loop = settings.loop || settings.rewind;

this._controls.$relative.toggleClass('disabled', !settings.nav || disabled);
```

**Comment by kiwiant** [comment]
Not fixed.

Definitely not related to the number of items as the problem occurs when there are 10 items with 3 being displayed and all initialised by the same selector/init code.

Seems to affect MS Edge 41.16299.15.0 with 2nd (of 2) carousels having `owl-nav disabled`. IE and Firefox didn't have this issue (race condition?).

**Comment by kiwiant** [comment]
My issue was occurred when a single item was loaded into the carousel with `loop:true`. The single item was repeated to fill carousel. For a listing of recent products this wasn't ideal so I had to set `loop:false`.

**Comment by gnxdev** [comment]
Yes, I can confirm that even when using custom navigation and `loop: true`, the event `next.owl.carousel` doesn't work.

If `loop: true` owl-carousel should always allow to go `next` and `prev` slides:
- With default navigation `nav: true` 
- Or via events `owl.trigger('next.owl.carousel')`, `owl.trigger('prev.owl.carousel')`

**Comment by ljjackson** [comment]
Has any progress been made on this?

**Comment by stickystuffssaikat** [comment]
#74  don’t know why, when I create
```
responsive: {
	  0: {
		items: 1
	  },
	  600: {
		items: 2
	  },
	  1000: {
		items: 5
	  }
	}
```
**nav** automatically **_disable_**
But when I create
```
         1000: {
		items: 4
	  }
```
It’s active again
By the way **both time** `nav: true,`
